src/status_im/contexts/wallet/add_account/create_account/import_private_key/view.cljs

```diff
@@ -112,7 +112,7 @@
         public-address              (rf/sub [:wallet/public-address])
         [flow-state set-flow-state] (rn/use-state nil)
         error?                      (= :invalid-private-key flow-state)]
-    (rn/use-unmount on-unmount)
+    (rn/use-mount on-mount)
```

Comment:
These now are the opposite, could you please elaborate on this?

Reply:
after the last changes - not required, removed